lib/verdict/experiment.rb

```diff
@@ -274,4 +249,12 @@ def ensure_experiment_has_started
   rescue Verdict::StorageError
     @started_at ||= Time.now.utc
   end
+
+  def fetch_assignment(subject)
+    @storage.retrieve_assignment(self, subject)
+  end
+
+  def fetch_subject(subject_identifier)
```

Comment:
oh sorry yeah i totally confused the. this should be public as originally, and just leave without implementation.